### PR TITLE
3rdparty: add jpgd project

### PR DESCRIPTION
**Pull request by tadanokojin**

### CODE REVIEW

File: pcsx2/pcsx2.vcxproj.filters [hunk 1]
```diff
@@ -533,9 +533,6 @@
     <ClCompile Include="IopMem.cpp">
       <Filter>System\Ps2\Iop</Filter>
     </ClCompile>
-    <ClCompile Include="IopSif.cpp">
-      <Filter>System\Ps2\Iop</Filter>
```
Comment (RedDevilus) [inline pcsx2/pcsx2.vcxproj.filters:537]: Isn't this still needed `iopsif`?

Comment (tadanokojin) [reply pcsx2/pcsx2.vcxproj.filters:537]: nope.

Comment (refractionpcsx2) [reply pcsx2/pcsx2.vcxproj.filters:537]: iopsif probably shouldn't even be there anymore.

File: pcsx2/pcsx2.vcxproj [hunk 2]
```diff
@@ -1172,10 +1168,13 @@
       <Project>{677b7d11-d5e1-40b3-88b1-9a4df83d2213}</Project>
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
     </ProjectReference>
+    <ProjectReference Include="..\3rdparty\jpgd\jpgd.vcxproj">
```
Comment (Florin9doi) [inline pcsx2/pcsx2.vcxproj:1171]:
```suggestion
    <ProjectReference Include="$(SolutionDir)3rdparty\jpgd\jpgd.vcxproj">
```

**Comment by tadanokojin** [comment]
@GovanifY is gonna follow up with CMake.

**Pull request merged**